deploy/firesim

```diff
@@ -125,9 +126,30 @@ def managerinit(args: argparse.Namespace):
             m = local("""cp sample-backup-configs/sample_config_{}.yaml config_{}.yaml""".format(conf_file, conf_file), capture=True)
             rootLogger.debug(m)
             rootLogger.debug(m.stderr)
-            m = local("""sed -i 's/AWSUSERNAME/{}/g' config_{}.yaml""".format(get_aws_userid(), conf_file), capture=True)
-            rootLogger.debug(m)
-            rootLogger.debug(m.stderr)
+
+            if args.platform == 'f1':
+                m = local("""sed -i 's/AWSUSERNAME/{}/g' config_{}.yaml""".format(get_aws_userid(), conf_file), capture=True)
+                rootLogger.debug(m)
+                rootLogger.debug(m.stderr)
+
+    rootLogger.info("Adding default overrides to default runtime.yaml file")
+    if args.platform == 'f1':
```

Comment:
To confirm, this is what people intended. I think this can be cleaned up later so that platform specific code doesn't leak into the general manager code.

Reply:
The mapping of platform values from args.platform to run farm recipe yamls seems weird, but I guess we can fix later.

Reply:
I'm not sure there is another way to implement this in the meantime... you need to know what run farm recipe corresponds to the platform that you setup for.

Reply:
I meant more that we call a single platform many different things:

- f1
- AWSEC2
- aws_ec2
...

We should probably pick one, but let's just fix it after this PR.